setup.py

```diff
@@ -103,16 +103,13 @@
 #      git tag v<VERSION> -m 'Adds tag v<VERSION> for pypi'
 #    Push the tag and release commit to git: git push --tags origin vXX.xx-release
 # 5. Run the following commands in the top-level directory:
-#      rm -rf dist
-#      rm -rf build
-#      python setup.py bdist_wheel
-#      python setup.py sdist
+#      make prepare_release
 # 6. Upload the package to the pypi test server first:
 #      twine upload dist/* -r testpypi
 # 7. Check that you can install it in a virtualenv by running:
 #      pip install accelerate
 #      pip uninstall accelerate
-#      pip install -i https://testpypi.python.org/pypi accelerate
+#      pip install -i https://test.pypi.org/simple/ accelerate
```

Comment:
What does this do? When I tried it locally, it downloaded metadata for all accelerate versions and then installed v0.0.1 :-O

Reply:
It should install it from the test pypi endpoint properly: https://test.pypi.org/project/accelerate/